### PR TITLE
✔️ Make instruction lists on channel/ticketer claim pages more consistent

### PR DESCRIPTION
**Pull request by rowanseymour**
Lot of different styles going on...

Captura de Pantalla 2020-11-17 a la(s) 16 00 46
Captura de Pantalla 2020-11-17 a la(s) 16 01 25
Captura de Pantalla 2020-11-17 a la(s) 16 01 37

So going with `.card` plus decimal lists..

Captura de Pantalla 2020-11-17 a la(s) 15 59 40
Captura de Pantalla 2020-11-17 a la(s) 15 59 48
Captura de Pantalla 2020-11-17 a la(s) 15 59 57
Captura de Pantalla 2020-11-17 a la(s) 16 00 09
Captura de Pantalla 2020-11-17 a la(s) 16 00 23

### CODE REVIEW

File: templates/channels/types/rocketchat/claim.haml [hunk 1]
```diff
@@ -5,25 +5,25 @@
   -blocktrans trimmed with name=brand.name
     You can connect Rocket.Chat to {{name}} in a few simple steps.
 
-  %ol.list-decimal.ml-5.mt-3
```
Comment (rowanseymour) [inline templates/channels/types/rocketchat/claim.haml:8]: stuff like this duplicated across lots of different files makes me think we should be trying harder to make reusable components from tailwind classes

**Comment by nicpottier** [comment]
Totally leave this to @ericnewcomer but does the card treatment make sense here? When are our rules around using cards?

**Comment by rowanseymour** [comment]
I'm not tied to any particular style - I think lists as cards was just the most common I saw

File: templates/channels/types/rocketchat/claim.haml (hunk 1 above)
Comment (ericnewcomer) [reply templates/channels/types/rocketchat/claim.haml:8]: The time to do this is when there are repeated styles across templates, so for example, list pages. Claim / Integration pages are another good candidate I would think.

**Review by ericnewcomer — APPROVED** [review]
Agree cards feel a bit out of place as instruction blocks, but at the same time I'm not totally offended by them and happy to have it be consistent. This of course is in lieu of a more thoughtful design across all of our claim pages.

**Comment by rowanseymour** [comment]
(not merging this for 6.0 because it's not a bugfix)

**Pull request merged**